lib/msal-electron-proof-of-concept/samples/msal-electron-sample-app/src/Main.ts

```diff
@@ -29,7 +30,10 @@ export default class Main {
         Main.mainWindow.on('closed', Main.onClose);
 
         // Once appplication is ready, configure MSAL Public Client App
-        Main.setupAuth();
+        Main.configureAuthentication();
+
+        // Listen for AcquireToken button call
+        Main.listenForAcquireToken();
```

Comment:
should this only be spun up when a token request is sent, or is it okay to always have open?

Reply:
OH i see nevermind